python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos_exploitation.py

```diff
@@ -1,27 +1,86 @@
 """
 Polynomial chaos exploitation
 =============================
+
+The goal of this example is to create a polynomial chaos expansion using the
+:class:`~openturns.FunctionalChaosAlgorithm` class and see the different methods
+of the :class:`~openturns.FunctionalChaosResult` class.
+In order to understand the different results, let us review some notations associated to the
+polynomial chaos expansion.
+
+We first introduce the physical input, output and model:
+- the random vector :math:`\\vect{X}` is the physical input random vector,
```

Comment:
```suggestion

- the random vector :math:`\\vect{X}` is the physical input random vector,
```
To make the bullet points appear